misc/scripts/refcnt_test.sh

```diff
@@ -80,20 +81,21 @@ function check_recovery_record {
 
 function test_crash_recovery {
     timeout=$1
-    echo "Checking recovery for VMDK plugin kill -9"
-    kill -9 `pidof docker-volume-vsphere`
-    until pids=$(pidof docker-volume-vsphere)
+    echo "Checking recovery through docker kill"
+    # kill docker daemon forcefully
+    pkill -9 dockerd
+    until pids=$(pidof dockerd)
```

Comment:
How does this impact the refcount module. Unless the plugin is restarted, the refcount module hasn't lost it's refcounts to do a recovery.